plugins/CMakeLists.txt

```diff
@@ -49,13 +49,10 @@ elseif("${CMAKE_SYSTEM_NAME}" MATCHES "FreeBSD")
 endif()
 
 if(WIN32 OR APPLE)
-	option(BUILD_BROWSER "Build browser plugin" OFF)
-	if (BUILD_BROWSER)
-		if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/obs-browser/CMakeLists.txt")
-			add_subdirectory(obs-browser)
-		else()
-			message(STATUS "obs-browser submodule not found!  Please fetch submodules.  obs-browser plugin disabled.")
-		endif()
+	if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/obs-browser/CMakeLists.txt")
+		add_subdirectory(obs-browser)
+	else()
+		message(WARNING "obs-browser submodule not found!  Please fetch submodules. obs-browser disabled")
```

Comment:
Nitpick:  There are two spaces after the first sentence, but there is only one space after the second sentence.  The original line used two spaces after each sentence, so use that.